copilot-bootstrap.py

```diff
@@ -438,6 +443,11 @@ def generate_storage(storage_config_file, output, overwrite):
 
     env_config = {}
 
+    click.echo("\n>>> Generating cloudformation\n")
+
+    # Validation TODO: check that the environments list matches what is in the copilot/environments/ dir
```

Comment:
Is this part of another ticket?

Reply:
Yep, service and env validation is done in the jsonschema PR

Reply:
Resolved in https://github.com/uktrade/copilot-tools/pull/27.